package.json

```diff
@@ -13,6 +13,7 @@
     "lint-fix": "fedx-scripts eslint --fix --ext .jsx,.js src/",
     "semantic-release": "semantic-release",
     "start": "fedx-scripts webpack-dev-server --progress",
+    "start-dev": "fedx-scripts webpack-dev-server --progress --host lmsbe.local.correlation-one.com",
```

Comment:
Does `--host` accept the argument without the "http://" ?

Reply:
yes it does 👍🏼